deploy/helm/Chart.yaml

```diff
@@ -39,16 +39,3 @@ dependencies:
 annotations:
   artifacthub.io/prerelease: "false"
   artifacthub.io/license: Apache-2.0
-  artifacthub.io/images: |
```

Comment:
This is not needed anymore and it disables automatic scan of images in ArtifactHub.